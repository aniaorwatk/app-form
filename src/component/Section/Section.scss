@import '../../scss/Main';

.main{
    display: flex;
  

}
.sectionBox{
  margin-left: auto;
    display: flex;
    flex-direction: column;

img{
    width: 10rem;
    height: 14vh;

}
.photo1{
    filter: grayscale(100%) brightness(70%) sepia(50%) hue-rotate(-100deg) saturate(500%) contrast(1);
}
.photo2{
    filter: grayscale(100%) brightness(70%) sepia(30%) hue-rotate(-100deg) saturate(500%) contrast(1);
}
.photo3{
    filter: grayscale(46%) sepia(90%) saturate(175%);
    -webkit-filter: grayscale(46%) sepia(100%) saturate(175%);
    -moz-filter: grayscale(46%) sepia(100%) saturate(175%); 
}
.photo4{
    filter: grayscale(46%) sepia(100%) saturate(175%);
    -webkit-filter: grayscale(46%) sepia(100%) saturate(175%);
    -moz-filter: grayscale(46%) sepia(100%) saturate(175%); 
}
.photo5{
    filter: grayscale(46%) sepia(100%) saturate(500%);
    -webkit-filter: grayscale(46%) sepia(100%) saturate(500%);
    -moz-filter: grayscale(46%) sepia(100%) saturate(500%);
}
}