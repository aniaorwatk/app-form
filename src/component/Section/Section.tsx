import React from "react";
import './Section.scss'