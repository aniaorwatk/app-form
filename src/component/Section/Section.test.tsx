import React from "react";
import { render } from "@testing-library/react";
import Section from "./Section";

describe("Section", () => {
  it("Should render", () => {
    const { container } = render(<Section />);
    expect(container).toMatchSnapshot();
  });
});

it('Should render img "Photo Albert Einstein" when src is okay', () => {
  const { getByAltText } = render(<Section />);
  const image = getByAltText("Photo Albert Einstein");
  expect(image).toHaveAttribute("src", "Albert-Einstein.jpg");
});

it('Should render img "Photo Dalajlama" when src is okay', () => {
  const { getByAltText } = render(<Section />);
  const image = getByAltText("Photo Dalajlama");
  expect(image).toHaveAttribute("src", "Dalajlama.jpg");
});

it('Should render img "Photo Henryk Sienkiewicz" when src is okay', () => {
  const { getByAltText } = render(<Section />);
  const image = getByAltText("Photo Henryk Sienkiewicz");
  expect(image).toHaveAttribute("src", "Henryk-Sienkiewicz.jpg");
});

it('Should render img "Photo Maria Skłodowska-Curie" when src is okay', () => {
  const { getByAltText } = render(<Section />);
  const image = getByAltText("Photo Maria Skłodowska-Curie");
  expect(image).toHaveAttribute("src", "Marie-Curie.jpg");
});

it('Should render img "Photo Wiliam Szekspir" when src is okay', () => {
  const { getByAltText } = render(<Section />);
  const image = getByAltText("Photo Wiliam Szekspir");
  expect(image).toHaveAttribute("src", "William-Szekspir.jpg");
});
