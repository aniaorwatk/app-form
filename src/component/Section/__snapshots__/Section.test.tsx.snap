// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Should render 1`] = `
<div>
  <div
    class="sectionBox"
  >
    <img
      alt="Photo Albert Einstein"
      class="photo1"
      src="Albert-Einstein.jpg"
    />
    <img
      alt="Photo Dalajlama"
      class="photo2"
      src="Dalajlama.jpg"
    />
    <img
      alt="Photo Henryk Sienkiewicz"
      class="photo3"
      src="Henryk-Sienkiewicz.jpg"
    />
    <img
      alt="Photo Maria SkÅ‚odowska-Curie"
      class="photo4"
      src="Marie-Curie.jpg"
    />
    <img
      alt="Photo Wiliam Szekspir"
      class="photo5"
      src="William-Szekspir.jpg"
    />
  </div>
</div>
`;
